load_testing/sp_ial2_sign_up.locustfile.py

```diff
@@ -0,0 +1,17 @@
+from locust import HttpUser, TaskSet, task, between
+from common_flows import flow_sp_ial2_sign_up, flow_helper
+
+
+class SP_IAL2_SignUpLoad(TaskSet):
+    # Preload drivers license data
+    license_front = flow_helper.load_fixture("mont-front.jpeg")
+    license_back = flow_helper.load_fixture("mont-back.jpeg")
```

Comment:
where are these used again?

Reply:
Those are used during the doc auth step here: https://github.com/18F/identity-loadtest/blob/main/load_testing/common_flows/flow_sp_ial2_sign_in.py#L144

Reply:
Oh weird, I didn't realize that class variables would be loaded on the instance like that, neat

Reply:
Made a lil test to confirm:

```python
class Foobar:
  license_front = 1


  def run(self):
    print(self.license_front)

x = Foobar()
x.run()
# prints 1
```

Reply:
I didn't either, I've been having a lot of fun learning python